Source/Fuse.Reactive.Expressions/BinaryOperator.uno

```diff
@@ -27,14 +27,23 @@ namespace Fuse.Reactive
 		protected virtual bool IsLeftOptional { get { return false; } }
 		protected virtual bool IsRightOptional { get { return false; } }
 
-		protected virtual object Compute(object left, object right)
+		protected virtual bool Compute(object left, object right, out object result)
 		{
-			throw new Exception(GetType().FullName + " does not implement the required methods");
+			Fuse.Diagnostics.Deprecated( " No `Compute`, or a deprecated form, overriden. Migrate your code to override the one with `bool` return. ", this );
+			result = Compute(left, right);
```

Comment:
I'm not sure this quite does the trick. What happens if a user overrides for instance BinaryFloatOperator, which already overloads this? Or NameValuePair? Or InfixOperator?

Reply:
Those already had their own set of virtual functions that shouldn't be affected by this I believe.

Reply:
I guess we should mark the Compute method as `sealed` on these, so the user can't override them, then.

Reply:
OK, seems like this is not as bad as I thought. We can do this to fix the theoretical problems in fuselibs:
```patch
diff --git a/Source/Fuse.Reactive.Expressions/MathFunctions.uno b/Source/Fuse.Reactive.Expressions/MathFunctions.uno
index 83c2067309..4dadcd52c2 100644
--- a/Source/Fuse.Reactive.Expressions/MathFunctions.uno
+++ b/Source/Fuse.Reactive.Expressions/MathFunctions.uno
@@ -152,7 +152,7 @@ namespace Fuse.Reactive
 			_name = name;
 			_op = op;
 		}
-		protected override object Compute(object operand)
+		protected sealed override object Compute(object operand)
 		{
 			float4 v;
 			int size;
@@ -193,7 +193,7 @@ namespace Fuse.Reactive
 			_name = name;
 			_op = op;
 		}
-		protected override object Compute(object left, object right)
+		protected sealed override object Compute(object left, object right)
 		{
 			double lv = 0;
 			double rv = 0;
diff --git a/Source/Fuse.Reactive.Expressions/NameValuePair.uno b/Source/Fuse.Reactive.Expressions/NameValuePair.uno
index 9460843397..30f5ed622f 100644
--- a/Source/Fuse.Reactive.Expressions/NameValuePair.uno
+++ b/Source/Fuse.Reactive.Expressions/NameValuePair.uno
@@ -3,7 +3,7 @@ using Uno.UX;
 namespace Fuse.Reactive
 {
 	/** Creates a `Fuse.NameValuePair` from a name and a value. */
-	public class NameValuePair: BinaryOperator
+	public sealed class NameValuePair: BinaryOperator
 	{
 		[UXConstructor]
 		public NameValuePair([UXParameter("Name")] Expression name, [UXParameter("Value")] Expression value) : base(name, value)
```